docs/json/sonarr/cf/uncensored.json

```diff
@@ -1,5 +1,6 @@
 {
   "trash_id": "026d5aadd1a6b4e550b134cb6c72b3ca",
+  "trash_score": "0",
```

Comment:
To my knowledge if a CF shouldn't get a score (0) then leave the score empty, Would need to ask @austinwbest if that's still the case,

Reply:
Spoke with Nit, you are correct have to remove the 0 scores